test/data/example_config.yaml

```
@@ -11,6 +11,13 @@ server:
     keyPath: "test/data/dummyServer.key"
     caPath: "test/data/dummyCa.pem"
     certRefreshPeriod: "24h"
+    disableCipherSuites:
```

@t4niwa 
Use PP for the list of disabled suites.
More natural variable name is important as these will be entered by end user.

Please apply this to every var you have introduced above.
```suggestion
    disabledCipherSuites:
```

---

I think present tense is more natural for config...🙈

---

@mlajkim 
Only can find a similar option in MS with present tense. All the others use the enable style. 😂
https://learn.microsoft.com/en-us/powershell/module/tls/disable-tlsciphersuite?view=windowsserver2022-ps

---

you are right. never mind @t4niwa

---

Resolve this please.

---

@mlajkim Fixed. Thank you.

---

found a PP form in jdk `jdk.tls.disabledAlgorithms` 🫠
https://aws.amazon.com/blogs/opensource/tls-1-0-1-1-changes-in-openjdk-and-amazon-corretto/

---

AWS > MS? lol.

---

I think both works then, and in that case its not necessary for changes. So I am good.
Honestly did not know until today because it sounds more natural to me using ```pp```.  

In short, good for resolve!